src/codegen.cpp

```diff
@@ -8292,15 +8292,18 @@ void add_cc_args(CodeGen *g, ZigList<const char *> &args, const char *out_dep_pa
         }
     }
 
+    //todo(dimenus): possibly break this up into separate lists
```

Comment:
I wouldn't consider this a TODO. Let's just explain the reasoning of why it is like this.